canonical-web-and-design/ubuntu.com
# Build Raspberry Pi server page

## Done

- Build page at /raspberry-pi/server

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/raspberry-pi/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc ](https://docs.google.com/document/d/19PHu9Rh9JB60C_RGb63hXLPypO67Vr6T4bLDWNPjQYg/edit#)


## Issue / Card

Fixes [#8474](https://github.com/canonical-web-and-design/ubuntu.com/issues/8474)
